pkg/apis/planetscale/v2/vitesscluster_types.go

```diff
@@ -135,6 +135,10 @@ type VitessClusterUpdateStrategy struct {
 	// Default: External
 	// +kubebuilder:validation:Enum=External,Immediate
 	Type *VitessClusterUpdateStrategyType `json:"type,omitempty"`
+
+	// ExternalOptions can optionally be used to enable the user to customize their external update strategy
+	// to allow certain updates to pass through immediately without using an external tool.
+	ExternalOptions *ExternalVitessClusterUpdateStrategyOptions `json:"external,omitempty"`
```

Comment:
The Go field name needs to match the JSON field name. Did `make generate` not complain about this?

```suggestion
	// External can optionally be used to customize the external update strategy.
	External *ExternalVitessClusterUpdateStrategyOptions `json:"external,omitempty"`
```

Reply:
`make generate` did not seem to complain, but I've changed it anyway. Done.